### PR TITLE
Updated health checks for content node to consume async status polling

### PR DESCRIPTION
**Pull request by dmanjunath**
This updates the file upload check from hearbeat to async status polling. This required some changes like the updated API signing object. The associated content node changes are at https://github.com/AudiusProject/audius-protocol/pull/3351.

This script has been used to test against a running content node.
```js
export delegatePrivateKey=...
export creatorNodeEndpoint=...
export spID=...


➜  sp-utilities git:(dm-updated-content-node-health-checks) ✗ node creator-node/healthChecks.js
web3-shh package will be deprecated in version 1.3.5 and will no longer be supported.
web3-bzz package will be deprecated in version 1.3.5 and will no longer be supported.
web3-shh package will be deprecated in version 1.3.5 and will no longer be supported.
web3-bzz package will be deprecated in version 1.3.5 and will no longer be supported.
Starting tests now. This may take a few minutes.
✓ Health check passed
✓ DB health check passed
✓ Disk health check passed
✓ Heartbeat duration health check passed
✓ Non-heartbeat duration health check passed
✓ File upload health check passed
All checks passed!
```

### CODE REVIEW

**Review by cheran-senthil — APPROVED** [review]

File: sp-utilities/creator-node/healthChecks.js [hunk 1]
```diff
@@ -162,8 +156,10 @@ async function healthCheckDuration () {
 
 // Test the file upload limit
 async function healthCheckFileUpload () {
-  const randomBytes = promisify(crypto.randomBytes)
-  let resp
+  if (!SP_ID) {
+    console.error('This cannot be run without a valid spID. If your node is not registered and does not have an spID yet, please contact a node operator with a registered ndoe to run this test')
```
Comment (vicky-g) [inline sp-utilities/creator-node/healthChecks.js:160]: `registered ndoe` -> `registered node`

File: sp-utilities/creator-node/healthChecks.js [hunk 2]
```diff
@@ -191,17 +183,85 @@ async function healthCheckFileUpload () {
...
+        spID: SP_ID,
+        timestamp,
+        signature
+      },
+      data: {
+        fileDir: TRANSCODE_STATE.data.resp.fileDir
+      },
+      responseType: 'json'
+    }
+    const clearArtifactsResp = await axios(clearRequestConfig)
```
Comment (vicky-g) [inline sp-utilities/creator-node/healthChecks.js:258]: wouldn't this throw if it doesn't 200? so we wouldnt see the thrown err on line 260

Comment (vicky-g) [reply sp-utilities/creator-node/healthChecks.js:258]: maybe wrap in try/catch if we want a particular err msg

**Review by vicky-g — COMMENTED** [review]
small comments

File: sp-utilities/creator-node/healthChecks.js (hunk 1 above)
Comment (dmanjunath) [reply sp-utilities/creator-node/healthChecks.js:160]: fixed

File: sp-utilities/creator-node/healthChecks.js (hunk 2 above)
Comment (dmanjunath) [reply sp-utilities/creator-node/healthChecks.js:258]: we actually don't need the explicit 200 check, axios will throw for non 2XX response codes so i'll just delete the if statement

**Pull request merged**